histolab/filters/image_filters.py

```diff
@@ -992,8 +995,11 @@ class OtsuThreshold(ImageFilter):
         Trans SystMan Cybern Syst 9.1 (1979)
     """  # noqa
 
+    def __init__(self, relate: Callable[..., Any] = operator.lt):
+        self.relate = relate
```

Comment:
why this has to be in the init and not directly in the caller?

Reply:
Because if we want to chain the filters, the only parameter in the caller must be the image/array.
We are doing the same with the other filters having additional parameters

Reply:
__gotcha__